checkpoint3/Program.cs

```diff
@@ -0,0 +1,257 @@
+        {
+            // todos_all.status[itemId] = "done";
+            context.todos.Find(itemId).status = true;
+        }
+        public void listPending()
+        {
+            Console.WriteLine("id | item\t\t\t| status");
+            Console.WriteLine("---+----------------------------+------------");
+            // for(int i = 0; i < todos_all.id.Count; i++)
+            // {
+            //     if(todos_all.status[i] == "pending")
+            //     {
+            //         Console.WriteLine(" {0}  | {1}                            | {2}", todos_all.id[i], todos_all.item[i], todos_all.status[i]);
+            //     }
+            // }
+            foreach(table aTable in context.todos)
+            {
+                if(aTable.status == false)
+                {
+                    Console.WriteLine(aTable);
+                }
+            }
+            
+        }
+        public void listDone()
+        {
+            Console.WriteLine("id | item\t\t\t| status");
+            Console.WriteLine("---+----------------------------+------------");
+            // for(int i = 0; i < todos_all.id.Count; i++)
+            // {
+            //     if(todos_all.status[i] == "done")
+            //     {
+            //         Console.WriteLine(" {0}  | {1}\t\t\t\t| {2}", todos_all.id[i], todos_all.item[i], todos_all.status[i]);
+            //     }
+            // }
+            foreach(table aTable in context.todos)
+            {
+                if(aTable.status == true)
+                {
+                    Console.WriteLine(aTable);
+                }
+            }
+        }
+        public void listAll()
+        {
+            Console.WriteLine("id | item\t\t\t| status");
+            Console.WriteLine("---+----------------------------+------------");
+            // for(int i = 0; i < todos_all.id.Count; i++)
+            // {
+            //     Console.WriteLine(" {0}  | {1}\t\t\t\t| {2}", todos_all.id[i], todos_all.item[i], todos_all.status[i]);
+            // }
+            foreach(table aTable in context.todos)
+            {
+                Console.WriteLine(aTable);
+            }
+        }
+        public table GetStudent(int id)
+        {
+            foreach(table aTable in context.todos)
+            {
+                if(aTable.id == id)
+                    return aTable;
+            }
+            return null;
+        }
+    }
+    public class Context : DbContext
+    {
+        public DbSet<table> todos {get; set;}
+        override
+        protected void OnConfiguring(DbContextOptionsBuilder optionsBuilder){
+            optionsBuilder.UseSqlite("Filename = ./todos.db");
+        }
+    }
+    // public class ConsoleUtils
+    // {
+    //     public static void print(List<table> toPrint)
+    //     {
+    //         foreach(table aTable in toPrint)
+    //         {
+    //             Console.WriteLine(aTable);
+    //         }
+    //     }
+    // }
+    public class table
```

Comment:
this class being called table is a little odd. and class names are usually capitalized.